thoth/storages/graph/postgres.py

```diff
@@ -5477,6 +5477,27 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                                 marker_evaluation_result=dependency.get("marker_evaluation_result", True),
                             )
 
+
+            # Sync imoprted package from result:tree
```

Comment:
this should run below https://github.com/thoth-station/storages/blob/e6fbbc34f782b8555729cf9f8fe270b2753f341f/thoth/storages/graph/postgres.py#L5332 :)

Reply:
resolved by 8e65f38da030758bc68904136a0c0888e7ed54aa